.vsts/builds/fast-pr-validation.yml

```diff
@@ -0,0 +1,24 @@
+trigger:
+- master
+- release/*
+
+# See https://github.com/aspnet/BuildTools
+resources:
+  repositories:
+  - repository: buildtools
+    type: github
+    endpoint: DotNet-Bot GitHub Connection
+    name: aspnet/BuildTools
+    ref: refs/heads/release/2.2
+
+phases:
+- template: .vsts-pipelines/templates/phases/default-build.yml@buildtools
+  parameters:
+    agentOs: Windows
+    buildArgs: "/t:CheckUniverse"
+
+phases:
+- template: .vsts-pipelines/templates/phases/default-build.yml@buildtools
+  parameters:
+    agentOs: Linux
```

Comment:
We don't have a macos agent?

Reply:
Good point, I'll add that. We had excluded macOS from PR validation in Travis in the past because their agents were too slow. The VSTS agents are much better.